module/zfs/vdev_draid.c

```diff
@@ -493,7 +493,11 @@ vdev_draid_generate_perms(const draid_map_t *map, uint8_t **permsp)
 	uint8_t *perms;
 
 	/* Allocate the permutation array */
+#ifdef	__dilos__
+	perms = kmem_alloc(permssz, KM_SLEEP);
+#else
 	perms = vmem_alloc(permssz, KM_SLEEP);
+#endif
```

Comment:
If `vmem_alloc()` and `vmem_free()` aren't defined for dilos then let's define `vmem_alloc/zalloc/free` to `kmem_alloc/zalloc/free`  in the same way it was done for FreeBSD.  This way we can keep the common code uncluttered with this kind of conditional code.